Assets/HoloToolkit/Common/Shaders/Standard.shader

```diff
@@ -522,7 +524,7 @@ Shader "MixedRealityToolkit/Standard"
                 {
                     int dataIndex = lightIndex * HOVER_LIGHT_DATA_SIZE;
                     fixed hoverValue = HoverLight(_HoverLightData[dataIndex], i.worldPosition.xyz, _HoverLightData[dataIndex + 1].a);
-                    pointToHover += hoverValue;
+                    pointToHover += (hoverValue * hoverValue);
```

Comment:
After reviewing this I think a good addition would be to make this feature "opt in." Much like how _HOVER_COLOR_OVERRIDE is done. That way existing materials will not change automatically on users when they get this change. Then I would change the shader parameter to something like _HoverLightRadiusOverride.